timeComplexity.js

```diff
@@ -0,0 +1,11 @@
+function containsDuplicates(arr) {
+    for (let i = 0; i < arr.length; i++) {
+      for (let j = 0; j < arr.length; j++) {
+        if (i !== j && arr[i] === arr[j]) {
+          return true;
+        }
+      }
+    }
+    return false;
+  }
```

Comment:
Maybe we can use a format like this:

### 🟠 Unnecessary time complexity
**Problem:** The containsDuplicates function has a time complexity of O(n^2), which can be optimized for better performance.

**Solution:** [describe the suggested solution]

```suggestion
function containsDuplicates(arr) {
    const set = new Set();
      for (let i = 0; i < arr.length; i++) {
        if (set.has(arr[i])) {
          return true;
        }
      set.add(arr[i]);
    }
    return false;
  }
```